### PR TITLE
Concat minified js files into one file

### LINKED ISSUE #175: Concat minified js files
Currently, we have script tags for `jquery.min.js`, `bootstrap.min.js`, etc. Instead we could concatenate these minified files into one file and use one script tag only.

### PR DESCRIPTION
**Pull request by elemoine**
Fixes #175.

### CODE REVIEW

File: CONST_Makefile [hunk 1]
```diff
@@ -171,6 +170,10 @@ geoportailv3/static/build/build.css: $(LESS_FILES) .build/node_modules.timestamp
 	mkdir -p $(dir $@)
 	./node_modules/.bin/lessc less/geoportailv3.less $@
 
+.build/build.js: build.json $(OL_JS_FILES) $(NGEO_JS_FILES) $(APP_JS_FILES) .build/externs/angular-1.3.js .build/externs/angular-1.3-q.js .build/externs/angular-1.3-http-promise.js .build/externs/jquery-1.9.js .build/node_modules.timestamp
+	mkdir -p $(dir $@)
```
Comment (sbrunner) [inline CONST_Makefile]: The .build must exists if .build/externs/angular-1.3.js exists :-)

Comment (elemoine) [inline CONST_Makefile]: Just to be sure :-)

File: CONST_Makefile [hunk 2]
```diff
@@ -159,9 +159,8 @@ geoportailv3/static/build/locale/lb/geoportailv3.json: geoportailv3/locale/lb/LC
 	mkdir -p $(dir $@)
 	node tasks/compile-catalog $+ > $@
 
-geoportailv3/static/build/build.js: build.json $(OL_JS_FILES) $(NGEO_JS_FILES) $(APP_JS_FILES) .build/externs/angular-1.3.js .build/externs/angular-1.3-q.js .build/externs/angular-1.3-http-promise.js .build/externs/jquery-1.9.js .build/node_modules.timestamp
-	mkdir -p $(dir $@)
-	node tasks/build.js $< $@
+geoportailv3/static/build/build.js: .build/node_modules.timestamp .build/build.js
+	awk 'FNR==1{print ""}1' $(addprefix node_modules/, jquery/dist/jquery.min.js angular/angular.min.js angular-gettext/dist/angular-gettext.min.js angular-animate/angular-animate.min.js bootstrap/dist/js/bootstrap.min.js) .build/build.js > $@
```
Comment (sbrunner) [inline CONST_Makefile]: I don't understand what's it will do ?
we can't simply do cat `$(addprefix ...) .build/build.js > $@` ?

Comment (sbrunner) [inline CONST_Makefile]: I just see that it will add a line brake between files, useful it it's missing :-)

Comment (elemoine) [inline CONST_Makefile]: And it was missing in some of the `min.js` files. That's why I added it.

File: CONST_Makefile [hunk 3]
```diff
@@ -159,9 +159,8 @@ geoportailv3/static/build/locale/lb/geoportailv3.json: geoportailv3/locale/lb/LC
 	mkdir -p $(dir $@)
 	node tasks/compile-catalog $+ > $@
 
-geoportailv3/static/build/build.js: build.json $(OL_JS_FILES) $(NGEO_JS_FILES) $(APP_JS_FILES) .build/externs/angular-1.3.js .build/externs/angular-1.3-q.js .build/externs/angular-1.3-http-promise.js .build/externs/jquery-1.9.js .build/node_modules.timestamp
-	mkdir -p $(dir $@)
-	node tasks/build.js $< $@
+geoportailv3/static/build/build.js: .build/node_modules.timestamp .build/build.js
```
Comment (elemoine) [inline CONST_Makefile]: A `mkdir` is missing here. Thank you Travis.

**Pull request merged**